railties/lib/rails/generators/app_base.rb

```diff
@@ -321,8 +321,11 @@ def javascript_gemfile_entry
           []
         else
           gems = [coffee_gemfile_entry, javascript_runtime_gemfile_entry]
```

Comment:
All this group should not be added. Neither turbolinks will work without sprockets.